spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/RabbitStreamMessageHandler.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2021-2022 the original author or authors.
```

Comment:
Just `2022`

Reply:
Well, strictly, 2021 is correct because this class was written in SCSt.

Reply:
Well, I didn't know the license works that way: it is really just a new class in this project, so its data is just actual, nothing more.